numbers/logSpace.go

```diff
@@ -37,12 +37,12 @@ func AddLog(x float64, y float64) float64 {
 	}
 	if x >= y {
 		if LogCanConvert(y - x) {
-			return x + math.Log(1+math.Exp(y-x))
+			return x + math.Log1p(math.Exp(y-x))
```

Comment:
What does "Log1p" do instead?

Reply:
I am not sure how the underlying approximation is different, but Log1p is a method in the standard library that has been specifically designed to more accurately handle taking the log of 1 plus a variable.  I believe the approximation is especially tuned for when the variable is small.  It looks like this does not noticeably change any of our test calculations, but is an improvement in theory.